src/index.ts

```diff
@@ -65,6 +70,12 @@ export interface RequestConfig {
   reqOpts: any;
   gaxOpts?: {};
 }
+export interface CreateInstanceRequest {
```

Comment:
Wouldn't it be better to let this interface extend the `IIinstance` interface, as they share almost all properties? This request only adds the `nodes` property.

Reply:
`IIinstance` interface provides misleading as well as unnecessary  properties for the purpose of `CreateInstance` function. I believe it is an error for `CreateInstanceRequest` interface to extend from `IIinstance` interface due to the following:

- `name` should not be part of the `config` param, it is passed in separately.
- `config` property is not optional.
- `nodeCount` should not be part of the `config` param. `nodeCount` is not sighted anywhere in the documentation and only erroneously sneaked in during typescript conversion when `CreateInstanceRequest` interface was set to extend `IIinstance` interface.
- `state` can not be set during instance creation, it is output only. 
- `endpointUris` can not be set during instance creation, it is output only.

Reply:
Once we set default value for `nodes`, as introduced in the PR, `nodes` parameter becomes optional.

Reply:
Sounds reasonable, thanks for the elaborate explanation.